### ISSUE TITLE
Make color palette colorblind-friendly

### ISSUE THREAD

**Issue by jamestamplin** [covid-projections/covid-projections#368]
*summary:*
Color blind / accessibility issues

*expected result:*

*actual result:*

*environment (device, browser, mobile / web):*

*link:*

*screenshot (you can copy/paste images directly in):*

**Comment by igorkofman** [comment]
I think we're pretty attached to the stoplight metaphor as a way of clearly communicating status, but I gather that there are ways of tweaking the colors to make them a lot more recognizable / seperable (by eg. adding blue to the green, perhaps in other ways).  Can someone make a suggestion on tweaks?

**Comment by andrewringler** [comment]
The current selection of colors on the homepage map is pretty close to being accessible, you have now: Red #F03147, Orange #F88722, Green #4FE6C1, Gray #E3E3E3

The Red/Orange/Green combination is fine, but the gray (of predictions not available) is too close to the green to be easily readable (for those who can only distinguish contrast well, not color, and for B&W printing).
See https://vis4.net/palettes/#/4|s|f03147,f88722,4fe6c1,e3e3e3|ffffe0,ff005e,93003a|0|0
![existing_colors](https://user-images.githubusercontent.com/691187/79020928-324d2600-7b48-11ea-9f0a-00289c7da7c1.png)

I would recommend just changing the color of Predictions not available to a darker color perhaps to: #fbfbfb

So you then have:
- Hospital overload in 0-4 weeks, Red #F03147
- Hospital overload in 4-8 weeks, Orange #F88722
- Hospital overload not likely, Green #4FE6C1
- Predictions not available, Gray #FBFBFB

See the new color accessibility analysis here: https://vis4.net/palettes/#/4|s|f03147,f88722,4fe6c1,fbfbfb|ffffe0,ff005e,93003a|0|0
![ColorProposa1](https://user-images.githubusercontent.com/691187/79021064-940d9000-7b48-11ea-9c1c-0691c7c416f7.PNG)

And on the map, the new colors look like this:
![NewColors_OnMap](https://user-images.githubusercontent.com/691187/79021130-c1f2d480-7b48-11ea-8210-1ad537159c19.png)

**Comment by thatneat** [comment]
The Chroma.js tool seems to be happy when I just change the gray color to a lighter gray `#fbfbfb` and leave everything else the same ( `#f03147`, `#f88722`, `#4be6bf` ).
The tool links provided above didn't work for me, but in case they do, here it is: https://vis4.net/palettes/#/4|s|f03147,f88722,4be6bf,fbfbfb|ffffe0,ff005e,93003a|0|0

![image](https://user-images.githubusercontent.com/3341011/79255608-691e8700-7e3b-11ea-81b2-809cb7e64e09.png)

I'll go ahead and submit a PR

**Comment by thatneat** [comment]
OK so the problem with the lighter grey is that it doesn't show up well against the background. 

Here's how it looks on Washington state:

![image](https://user-images.githubusercontent.com/3341011/79260217-c36f1600-7e42-11ea-9400-3c194a06c7b3.png)

several counties are all but invisible. Even the darker, `#e3e3e3` gray we're using now, doesn't meet WCAG guidelines for contrast ratio in graphics.

In @andrewringler 's screenshot above, there are visible borders between the counties which makes it work, but that doesn't currently seem to be implemented. Does anyone have context on those?

**Comment by thatneat** [comment]
`#00ff00`
`rgba(0,0,0,1)`

**Comment by thatneat** [comment]
A better way to look at the color scheme is to add the background color. That makes it clear that the current gray is already close to the lightness midpoint of the green and white colors. Making the gray any lighter only moves it closer to the white and causes a problem there. The palette checker considers any combination of grays and white is considered "colorblind-safe" which I think is a bug.

![image](https://user-images.githubusercontent.com/3341011/79264621-bace0e00-7e49-11ea-8ed5-3d062471f216.png)

That leaves us with a couple of options:

1. add borders:

![image](https://user-images.githubusercontent.com/3341011/79264955-392ab000-7e4a-11ea-830c-dc07c39b0f59.png)
![image](https://user-images.githubusercontent.com/3341011/79266258-244f1c00-7e4c-11ea-9ff6-0d3b46179a0d.png)
![image](https://user-images.githubusercontent.com/3341011/79266613-b0f9da00-7e4c-11ea-98f0-a9776e9ad190.png)

Note that the mini-legend at the bottom becomes a little harder to understand :(

2. make the gray darker, say `#aaaaaa`:
![image](https://user-images.githubusercontent.com/3341011/79265126-78590100-7e4a-11ea-94f2-b11c949312dc.png)

![image](https://user-images.githubusercontent.com/3341011/79265242-ab9b9000-7e4a-11ea-9a4f-9ddf27af1d50.png)

#2 may be better, but #1 is a smaller visual change so I'll put up a PR for that one unless anyone objects.

**Comment by andrewringler** [comment]
@thatneat oh, I was quite confused for sometime, until I realized your screenshot of Washington state is actually the thumbnail from here:
<https://covidactnow.org/us/wa>

On the homepage, the U.S. state map already does have borders for counties `rgba(255,255,255,0.05)` and for states `rgba(0,0,0,0.15)`

I agree with using something like your option #1. Make `Predictions not available` a light color, and add add dark county and state borders on the state pages, and make sure the U.S. map on the homepage has all the same colors and borders.

**Comment by thatneat** [comment]
Great, that's basically what I just posted in #533 .

There are slight differences in the borders between the state map and the USA map because the USA map also has state boundaries, and the zoom levels are different so things generally look different.
